### ISSUE TITLE
not seeing the pico remotes in HomeKit

### ISSUE THREAD

**Issue by 1564MAT** [thenewwazoo/homebridge-lutron-caseta-leap#107]
Hello, I am not seeing the pico remotes in HomeKit. I see them in homebridge. Can someone help fix the issue?

**Comment by thenewwazoo** [comment]
Hi @1564MAT, I'm sorry to see you were having problems. Unfortunately, resolving this is outside of what I can help with. I had hoped another user would speak up, but nobody did. If you find any clues, please attach a new comment sharing what you learn!

**Comment by 1564MAT** [comment]
Hello Brandon. Thanks for following up. I should have erased the issue. After a few weeks , I realized there was a default room that has been created in my home. Since the device is in my kitchen, I was looking for it it my HomeKit kitchen room. This is why I posted the issue. Sorry about it and thanks for the great plug-in. I love it !  Let me know if I can help. MatLe 21 mai 2023 à 12:29, Brandon Matthews ***@***.***> a écrit :﻿
Closed #107 as completed.
